src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+                        break;
+                    case "list":
+                        printList(lists, lists.size());
+                        break;
+                    case "mark":
+                        lineOne();
+                        System.out.println("Good Job! ^_^");
+                        System.out.println("Task number " + textSplit[1] + " has been marked as done!");
+                        int tNum = Integer.parseInt(textSplit[1]);
+                        lists.get(tNum - 1).done();
+                        System.out.println(lists.get(tNum - 1));
+                        lineOne();
+                        break;
+                    case "unmark":
+                        lineOne();
+                        System.out.println("I've unmarked task number " + textSplit[1]);
+                        System.out.println("Complete it soon! ^_^");
+                        int tNo = Integer.parseInt(textSplit[1]);
+                        lists.get(tNo - 1).undo();
+                        System.out.println(lists.get(tNo - 1));
+                        lineOne();
+                        break;
+                    case "delete" :
+                        lineOne();
+                        System.out.println("Noted. I've removed this task:");
+                        //System.out.println("Complete it soon! ^_^");
+                        int t2No = Integer.parseInt(textSplit[1]);
+                        System.out.println(lists.get(t2No - 1));
+                        lists.remove(t2No - 1);
+                        lineOne();
+                        break;
+                    case "todo":
+                        try {
+                            if(fullDesc.equals(" ") || fullDesc.equals("")) {
+                                throw new DukeInvalidTodo();
+                            }
+                        }
+                        catch(DukeInvalidTodo e) {
+                            System.err.println(e.getMessage());
+                            continue;
+                        }
+                        lineTwo();
+                        System.out.println("New task added:");
+                        Task t = new ToDo(fullDesc);
+                        lists.add(t);
+                        System.out.println(t);
+                        System.out.println("You have " + lists.size() + " tasks left now! ^_^");
+                        lineTwo();
+                        break;
+                    case "event":
+                        lineTwo();
+                        System.out.println("New task added:");
+                        Task t1 = new Event(fullDesc, textSplitOne[1]);
+                        lists.add(t1);
+                        System.out.println(t1);
+                        System.out.println("You have " + lists.size() + " tasks left now! ^_^");
+                        lineTwo();
+                        break;
+                    case "deadline":
```

Comment:
Logical units within a block should be separated by a line, perhaps a space after each case would be ideal!